engine/src/test/java/org/hibernate/validator/test/internal/engine/valuehandling/OptionalWithPrivateGetterTest.java

```diff
@@ -41,7 +42,7 @@ public void testConstraintCanBePlacedOnPrivateGetter() {
 	public void testConstraintOnPrivateGetterReturnsConstraintViolation() {
 		Validator validator = getValidator();
 
-		LocalDate[][] testData = { { LocalDate.MAX, LocalDate.MIN }, { LocalDate.now(), null }, { null, null } };
+		LocalDate[][] testData = { { LocalDate.MAX, LocalDate.MIN }, { LocalDate.now( ZoneId.systemDefault() ), null }, { null, null } };
```

Comment:
I'm a bit sceptic about this type of changes. @gunnarmorling what's your take on this? I'm not worried about these occurrences but more about how it will be frustrating when we write new tests.